eventkit_cloud/utils/osmparse.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 import argparse
 import logging
+from os.path import exists
 import os
```

Comment:
There are other os features used; the special import of exists is to make mocking it specifically easier in the tests.